packages/react-avatar/src/components/AvatarGroup/AvatarGroup.types.ts

```diff
@@ -1,19 +1,72 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { PopoverSurface } from '@fluentui/react-popover';
+import { AvatarSizes } from '../Avatar/Avatar.types';
+import { Button } from '@fluentui/react-button';
+import { TooltipProps } from '@fluentui/react-tooltip';
 
 export type AvatarGroupSlots = {
   root: Slot<'div'>;
+
+  /**
+   * Popover trigger slot that can be used to change the overflow indicator.
+   */
+  popoverTrigger?: Slot<typeof Button>;
+
+  /**
+   * Popover surface that will be displayed when the popover is triggered.
+   */
+  popoverSurface?: Slot<typeof PopoverSurface>;
 };
 
 type AvatarGroupCommons = {
-  // TODO Add things shared between props and state here
+  /**
+   * Layout the Avatars should be displayed as.
+   * @default grid
+   */
+  layout: 'grid' | 'stacked' | 'pie';
```

Comment:
I don't really like "grid" for the layout described in the spec. When I hear "grid" I think of a two-dimensional layout and that's not the case here, rather it's a one-dimensional row of avatars.

What do you think about `inline` instead of `grid`?